static/bidder-info/lemmadigital.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: "https://sg.ads.lemmatechnologies.com/lemma/servad?pid={PID}&aid={AID}"
```

Comment:
@lm-ved how are `pid` and `aid` used ?

Reply:
They are used for specific targeting. These values are communicated by the team before integration can go ahead.
[Doc link.](https://github.com/prebid/prebid.github.io/blob/fd9d87a97901b26c15962427861c321b6f161a08/dev-docs/bidders/lemmadigital.md?plain=1#L24)